contracts/payloads/IGP63/PayloadIGP63.sol

```diff
@@ -0,0 +1,397 @@
+        INST_ETH_VAULT_ID = inst_eth_vault_id;
+        ETH_USDC_DEX_ID = eth_usdc_dex_id;
+        ETH_USDC_VAULT_ID = eth_usdc_vault_id;
+    }
+
+    /**
+     * |
+     * |     Proposal Payload Actions      |
+     * |__________________________________
+     */
+
+    /// @notice Action 1: Increase weETH-ETH_wstETH limits
+    function action1() internal {
+        address WEETH_ETH_WSTETH_DEX_ADDRESS = getDexAddress(9);
+        address WEETH_ETH_WSTETH_VAULT_ADDRESS = getVaultAddress(74);
+
+        { // Increase WEETH-ETH-WSTETH vault limit
+            IFluidAdminDex.UserSupplyConfig[] memory config_ = new IFluidDex.UserSupplyConfig[](1);
+            config_[0] = IFluidAdminDex.UserSupplyConfig({
+                user: WEETH_ETH_WSTETH_VAULT_ADDRESS,
+                expandPercent: 25 * 1e2, // 25%
+                expandDuration: 12 hours, // 12 hours
+                baseWithdrawalLimit: 6_000 * 1e18 // 6k shares
+            });
+
+            IFluidDex(WEETH_ETH_WSTETH_DEX_ADDRESS).updateUserSupplyConfigs(
+                config_
+            );
+        }
+
+        {
+            BorrowProtocolConfig memory config_ = BorrowProtocolConfig({
+                protocol: WEETH_ETH_WSTETH_VAULT_ADDRESS,
+                borrowToken: wstETH_ADDRESS,
+                expandPercent: 20 * 1e2, // 20%
+                expandDuration: 12 hours, // 12 hours
+                baseBorrowLimitInUSD: 42_000_000, // $42M
+                maxBorrowLimitInUSD: 92_000_000 // $92M
+            });
+
+            setBorrowProtocolLimits(config_);
+        }
+    }
+
+    /// @notice Action 2: Update fUSDC and fUSDT Rewards
+    function action2() internal {
+        address[] memory protocols = new address[](2);
+        address[] memory tokens = new address[](2);
+        uint256[] memory amounts = new uint256[](2);
+
+        address fSTABLES_REWARDS_ADDRESS = 0xb75Ec31fd7ad0D823A801be8740B9Fad299ce6d6;
+
+        {
+            /// fUSDC
+            IFTokenAdmin(F_USDC_ADDRESS).updateRewards(
+                fSTABLES_REWARDS_ADDRESS
+            );
+
+            uint256 allowance = IERC20(USDC_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDC_ADDRESS
+            );
+
+            protocols[0] = F_USDC_ADDRESS;
+            tokens[0] = USDC_ADDRESS;
+            amounts[0] = allowance + (500_000 * 1e6);
+        }
+
+        {
+            /// fUSDT
+            IFTokenAdmin(F_USDT_ADDRESS).updateRewards(
+                fSTABLES_REWARDS_ADDRESS
+            );
+
+            uint256 allowance = IERC20(USDT_ADDRESS).allowance(
+                address(FLUID_RESERVE),
+                F_USDT_ADDRESS
+            );
+
+            protocols[1] = F_USDT_ADDRESS;
+            tokens[1] = USDT_ADDRESS;
+            amounts[1] = allowance + (500_000 * 1e6);
+        }
+
+        FLUID_RESERVE.approve(protocols, tokens, amounts);
+
+        ILendingRewards(fSTABLES_REWARDS_ADDRESS).start();
```

Comment:
no need to call for this one, it will auto-transition from previous model when they run out on ~Dec20